git-extra/git-update-git-for-windows

```diff
@@ -162,25 +162,23 @@ update_git_for_windows () {
 		;;
 	esac
 
-	releases_url=https://api.github.com/repos/git-for-windows/git/releases
-	releases=$(http_get $releases_url/latest) ||
+	latest_tag_url=https://gitforwindows.org/latest-tag.txt
+	latest_tag=$(http_get $latest_tag_url) ||
 	case $?,"$proxy" in
 	7,)
-		proxy="$(proxy-lookup.exe https://api.github.com)" &&
+		proxy="$(proxy-lookup.exe https://gitforwindows.org)" &&
 		test -n "$proxy" &&
 		export https_proxy="$proxy" &&
 		echo "Using proxy $https_proxy as per lookup" >&2 &&
-		releases=$(http_get $releases_url/latest) ||
+		latest_tag=$(http_get $latest_tag_url) ||
 		return
 		;;
 	*)
 		return
 		;;
 	esac
 
-	latest=$(echo "$releases" |
-		grep '"tag_name": "v' |
-		sed -E 's/.*"tag_name": "v([^"]*).*/\1/')
+	latest=$(echo "$latest_tag" | sed s/^v//)
```

Comment:
How about `latest=${latest_tag#v}`? That should be a lot quicker, as it avoids spawning a new process.